bftengine/include/bcstatetransfer/SimpleBCStateTransfer.hpp

```diff
@@ -76,6 +77,29 @@ class IAppState {
   // outBlock and outBlockSize. Returns true IFF block blockId exists.
   virtual bool getBlock(uint64_t blockId, char *outBlock, uint32_t *outBlockSize) = 0;
 
+  struct GetBlockContext {
+    // input
+    uint64_t blockId;
+
+    // output
+    uint64_t jobDurationMicrosec;
```

Comment:
Sound OK for me, I'll use std::chrono directly, DuratiinTracker doesn't 't support that.
I can make a new function returning T, but this will require more changes in this class, which I don't see as necessary for now.